src/components/elements/GatewayCard.tsx

```diff
@@ -88,21 +88,19 @@ const GatewayCardComponent = (props: GatewayProps) => {
         </Col>
       </Row>
 
-      <h2 data-testid="sensor-header" className={styles.sectionSubTitle}>
-        Sensors
+      <h2 data-testid="node-header" className={styles.sectionSubTitle}>
+        Nodes
       </h2>
-      {gatewayDetails.sensorList.length ? (
+      {gatewayDetails?.nodeList.length ? (
```

Comment:
```suggestion
      {gatewayDetails.nodeList.length ? (
```

Reply:
I didn't see a TS error that gatewayDetails.nodeList might be undefined (I think it will always exist now as an empty array, at least), so you can probably remove this optional chaining

Reply:
Good call. Not sure why I added this. Might’ve been to work around a bad name or something. Fixed 👍